import-export-cli/specs/params/params.go

```diff
@@ -31,12 +31,20 @@ type Configuration struct {
 	ActionDuration *int `yaml:"actionDuration,omitempty" json:"actionDuration,omitempty"`
 }
 
+// Advance endpoint config for MGW
+type AdvanceConfigForMG struct {
```

Comment:
If this is not MGW specific, shall we rename it to `AdvanceConfig`?

Reply:
Updated with https://github.com/wso2/product-apim-tooling/pull/991/commits/7b8a37ff90f6cba199f858b0715a6a0eca5d7748